src/rascal/structure_managers/property_typed.hh

```diff
@@ -319,7 +319,11 @@ namespace rascal {
      * Accessor for last pushed entry for dynamically sized properties
      */
     reference back() {
-      auto && index{this->values.size() - this->get_nb_comp()};
+      // -1 is correction for 0-start indexing
```

Comment:
The comment should move next to the `index = ...` line

Reply:
Sure, I'll wait for the CI to pass again.